resources/schema/USDM.yaml

```diff
@@ -0,0 +1,73 @@
+"`this`": Wrapper
+ACTIVITY: ScheduledActivityInstance
+activities: Activity
+activityIds: Activity
+analysisPopulation: AnalysisPopulation
+bcSurrogateIds: BiomedicalConceptSurrogate
+bcSurrogates: BiomedicalConceptSurrogate
+code: Code
+codes: Code
+country: Code
+DECISION: ScheduledDecisionInstance
+dataOriginType: Code
+definedProcedures: Procedure
+ELIGIBILITY_CRITERIA: EligibilityCriteria
+encounterContactModes: Code
+encounterEnvironmentalSetting: Code
+encounters: Encounter
+ENDPOINT: Endpoint
+endpointLevel: Code
+epochId: StudyEpoch
+GEOGRAPHIC_SCOPE: GeographicScope
+intercurrentEvents: IntercurrentEvent
+interventionModel: Code
+nextActivityId: Activity
+nextEncounterId: Encounter
+nextStudyEpochId: StudyEpoch
+OBJECTIVE: Objective
+objectiveEndpoints: Endpoint
+objectiveLevel: Code
+organizationLegalAddress: Address
+plannedSexOfParticipants: Code
+previousActivityId: Activity
+previousEncounterId: Encounter
+previousStudyEpochId: StudyEpoch
+protocolStatus: Code
+scheduleTimelineExits: ScheduleTimelineExit
+scheduledActivityInstanceEncounterId: Encounter
+timings: Timing
+standardCode: Code
+study: Study
+studyArmId: StudyArm
+studyArms: StudyArm
+studyCells: StudyCell
+studyDesignBlindingScheme: Code
+studyDesigns: StudyDesign
+studyElementIds: StudyElement
+studyElements: StudyElement
+studyEpochId: StudyEpoch
+studyEpochs: StudyEpoch
+studyEstimands: Estimand
+studyIdentifierScope: Organization
+studyIdentifiers: StudyIdentifier
+studyIndications: Indication
+studyInterventions: StudyIntervention
+studyObjectives: Objective
+studyPhase: AliasCode
+studyPopulations: PopulationDefinition
+studyProtocolVersions: studyProtocolVersions
+studyScheduleTimelines: ScheduleTimeline
+SUBJECT_ENROLLMENT: SubjectEnrollment
+timelineId: ScheduleTimeline
+timingRelativeToFrom: Timing
```

Comment:
This should be `timingRelativeToFrom: Code`

Reply:
Updated